core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/tenant/JDBCTenantManager.java

```diff
@@ -905,6 +910,23 @@ public void deleteTenant(int tenantId, boolean removeFromPersistentStorage)
         }
     }
 
+    /**
+     * Delete all tenant information related to tenant stored in UM tables.
+     *
+     * @param tenantId Id of the tenant
+     * @throws org.wso2.carbon.user.api.UserStoreException
```

Comment:
Shall we add only the UserStoreException rather than the relative path? WDYT?

Reply:
The full path is included here as there is another `UserStoreException` which is inherited by this. But I think we can use that Exception here.
+1